default.nix

```diff
@@ -435,12 +435,13 @@ in let this = rec {
   inherit cabal2nixResult system androidSupport iosSupport;
   project = args: import ./project this (args ({ pkgs = nixpkgs; } // this));
   tryReflexShell = pinBuildInputs ("shell-" + system) tryReflexPackages;
-  ghcjsExternsJs = ./ghcjs.externs.js;
 };
 
 # Deprecated reexports. These were made for `./scripts/*`, but are reexported
 # here for backwards compatability.
 legacy = {
+  # Added 2020-03, will be removed 2020-09.
+  ghcjsExternsJs = ./legacy/ghcjs.externs.js;
```

Comment:
This is used by obelisk:

https://github.com/obsidiansystems/obelisk/blob/0133449c597afdb3d4aaeb7a6475610b7bca6c99/default.nix#L164

Reply:
Oh! I'll back out of that commit for now then. Thanks!